client/dashboard/components/inline-support-link/index.tsx

```diff
@@ -0,0 +1,63 @@
+import { ExternalLink } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import React from 'react';
+import useSupportDocData from 'calypso/components/inline-support-link/use-support-doc-data'; // eslint-disable-line no-restricted-imports
```

Comment:
Curious why not just add this to the .eslintrc.js?

Reply:
I'd prefer to import it only in this file. It shouldn't be used elsewhere in Dashboard V2.